programs/runtests.c

```diff
@@ -14,86 +14,95 @@
 #include <sys/unistd.h>
 #include <sys/wait.h>
 
+// Setup the logging for this file (do this before any other include).
+#include "sys/kernel_levels.h"         // Include kernel log levels.
+#define __DEBUG_HEADER__ "[TRUNS ]"    ///< Change header.
+#define __DEBUG_LEVEL__  LOGLEVEL_INFO ///< Set log level.
+#include "io/debug.h"                  // Include debugging functions.
+
 #define SHUTDOWN_PORT 0x604
 /// Second serial port for QEMU.
 #define SERIAL_COM2 0x02F8
 
 static char *all_tests[] = {
-    "t_abort",
-    "t_alarm",
-    "t_creat",
-    "t_dup",
-    "t_exec execl",
-    "t_exec execlp",
-    "t_exec execle",
-    "t_exec execlpe",
-    "t_exec execv",
-    "t_exec execvp",
-    "t_exec execve",
-    "t_exec execvpe",
-    "t_fork 10",
-    "t_gid",
-    "t_groups",
-    "t_itimer",
-    "t_kill",
+    // "t_abort",
```

Comment:
Why disabling all tests?

Reply:
Sorry, my bad, I was disabling them to focus on write/read from file.